book/website/collaboration/ppie-events/ppie-events-location.md

```diff
@@ -0,0 +1,74 @@
+Consider how you will direct and guide people at the venue. If you plan to have banners that will displayed near parking and transport links that can direct attendees to the event registration, will you also have staff members who will be able to guide attendees with low vision or blindness.
+
+Consider where will you put signs at the venue to indicate where the sessions are taking place, and also where any facilities such as restrooms, elevators, kitchen space, and all breakout sessions are located. Signs should be clearly visible and identifiable to your attendees. 
+Try to place signs and information before **and** at junctions so that people do not take a wrong turn down a corridor, and if there are locations at your venue that may be difficult to navigate, consider having staff visible to help direct attendees. Use directional arrows, and have signs distributed regularly along paths. 
+
+Although they may help some individuals, remember that signs and visual information may not be accessible to all attendees and consider how to provide clear guidance at the venue to attendees with visual impairments. 
+When assisting a disabled person, ask them how you can do this because it varies between individuals.
+Do not touch, grab, or move people with disabilities, their mobility aids, or service dogs without permission.
+
+It is also important to be cognizant that attendees who are more familiar with the layout of University campuses may feel more comfortable navigating university-based locations. 
+Please consider how you will make members of the public feel comfortable in these spaces. 
+This may mean putting up posters or banners and decorating areas of the space to allow attendees to feel a sense of belonging and identifying where your event spaces are located within a larger venue.
+
+If you are able to provide attendees with a map of the venue, make sure to identify how to navigate the location, where parking and restrooms are located, where to find staff members, and where all important locations will be (such as kitchen, break out rooms, and registration). 
+Also ensure that the location of any emergency supplies such as an AED, epi-pen, first aid kit and a member of staff who will be able to act in the event of an emergency is marked on the map. 
+Although it is not expected for staff and organizers to be medically trained, it is important to know who needs to be contacted in an emergency.
+
+(cl-ppie-events-location-setup)=
+## Setting up the event
+If your event is held at a venue where you are unable to alter the seating layout, avoid dedicating an entire row or completely separate seating area for these attendees. 
+Segregated seating and single areas can make attendees with lived experience feel separate from their colleagues and singled out. 
+By distributing the seating throughout a space you can create a more inclusive and welcoming environment for attendees. 
+
+
+(cl-ppie-events-location-during-event)=
+## During the event
```

Comment:
Are sensory maps something that you think should be added to this section? I've seen them commonly used in making more spaces friendly for autistic participants? What do you think?

Maybe: Consider creating a sensory map, which is designed to help attendees understand and plan for the sensory experiences they may have throughout the space.

For example: https://www.sciencemuseum.org.uk/sites/default/files/2022-07/Sensory-Map-Science-Museum-July-2022.pdf

Reply:
I had left those suggestions to the accessibility chapter :)